client/blocks/comment-detail/comment-detail-edit.jsx

```diff
@@ -85,13 +94,29 @@ export class CommentDetailEdit extends Component {
 				</FormFieldset>
 
 				<FormTextarea
+					disabled={ ! isEditCommentSupported }
 					onChange={ this.setCommentContentValue }
 					value={ commentContent }
 				/>
 
+				{ ! isEditCommentSupported &&
+					<Notice
+						status="is-warning"
+						showDismiss={ false }
+						text={ translate(
+							'Comment editing requires a newer version of Jetpack.'
+						) }
+					>
+						<NoticeAction href={ `/plugins/jetpack/${ siteSlug }` }>
+							{ translate( 'Update Now' ) }
```

Comment:
I think it'll be cool to update Jetpack in place using `PluginActions` like: https://github.com/Automattic/wp-calypso/blob/78ee303a91bf5fe571c315bfb552c5b946144b66/client/my-sites/plugins/plugin-meta/index.jsx#L389

It's a lot more complicated (maybe for **M3**), but considering number of actions the user has to take to reach this notice, it may be nice to prevent it from leaving the page.
Not a blocker, but something to think about.

Reply:
I wasn't aware of such thing!
Let's open an issue on M2, and if it's _that_ complicated we can move it to M3, since it's not a blocker whatsoever. 🙂